guess-the-country/country-frontend/src/components/Score.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="wrapper">
- 
-   <section v-if="(this.user_city_answer && this.round==11)" class="left xd-section xd-light">
+    <!-- remove "+1" to show text --->
```

Comment:
?

Reply:
Wir wollen, dass (zunächst) die Nachricht "Now you can finish the game!" nicht angezeigt wird. Deshalb kurzerhand die +1, sodass this.round niemals this.nr_of_rounds wird.

Reply:
Dann lösch doch einfach den Teil, den du nicht haben willst :-) Solche Hacks und Kommentare sind nie gut und führen nur zur Verwirrung.